tools/pipelines/templates/build-npm-package.yml

```diff
@@ -560,44 +560,45 @@ stages:
         isReleaseGroup: ${{ parameters.isReleaseGroup }}
         publishNonScopedPackages: ${{ variables.publishNonScopedPackages }}
 
-  # Capture per-pipeline stage results
-  - stage: upload_run_telemetry
-    displayName: Upload pipeline run telemetry to Kusto
-    condition: and(succeededOrFailed(), eq(variables['Build.Reason'], 'IndividualCI'), eq(${{parameters.telemetry}}, true))
-    dependsOn:
-      - build
+  # Capture pipeline stage results
+  - ${{ if eq(parameters.telemetry, true) }}:
+    - stage: upload_run_telemetry
+      displayName: Upload pipeline run telemetry to Kusto
+      condition: succeededOrFailed()
```

Comment:
This indentation change is interesting. I'm surprised the previous indentation didn't cause issues.

Reply:
I was also surprised. YAML parsing and validation feels like black magic every time we see one of these.